apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -230,9 +230,11 @@ const EVENTS = {
   EXPORT_APP: 'User Exports App From Share Advanced Options',
 
   // Curriculumm Recommender
-  RECOMMENDED_SIMILAR_CURRICULUM_SHOWN: 'Recommended Similar Curriculum Shown',
+  RECOMMENDED_CATALOG_CURRICULUM_SHOWN: 'Recommended Catalog Curriculum Shown',
```

Comment:
tagging @dmcavoy as an fyi that this is changing. Now sure if you're looking at this metric right now